docs/products/flink/overview.rst

```diff
@@ -0,0 +1,97 @@
+Service overview
```

Comment:
This documentation does not fit the diataxis model, let's take some time and move the various sections into the existing structure to make them easier to find (I can help with this)

Reply:
I'd prefer to keep this as a single article, as it's intended to provide a summary of what's included in the service in one place - but I suppose it could be moved under concepts (done in https://github.com/aiven/devportal/pull/349)